tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -1309,6 +1326,11 @@ def _serialize_and_track_engine(node):
 
       filename = os.path.join(engine_asset_dir,
                               "trt-serialized-engine." + canonical_engine_name)
+      if self._need_calibration and self.calib_tbl_asset_dir != None:
```

Comment:
I think this should be:
    if self._need_calibration:
      assert self.calib_tbl_asset_dir != None

Am I right? If not, in what situation we have self._need_calibration == true and self.calib_tbl_assert_dir == None?

Reply:
Yes I think you're right.